core/src/main/java/edu/wpi/grip/core/Operation.java

```diff
@@ -23,6 +31,13 @@
     String getDescription();
 
     /**
+     * @return What category the operation falls under.  This is used to organize them in the GUI
+     */
+    default Category getCategory() {
+        return Category.MISCELLANEOUS;
+    }
```

Comment:
I can actually remove this.  I added it at first, but I think we generally want to specify it explicitly.